ingest/importer/submission/entity_map.py

```diff
@@ -0,0 +1,120 @@
+from typing import List
+
+from ingest.importer.submission.entity import Entity
+
+
+class EntityMap(object):
+
+    def __init__(self, *entities):
+        self.entities_dict_by_type = {}
+        if entities is not None:
```

Comment:
consider reversing the predicate to decrease levels of indentation.

Reply:
Won't touch this file for now.

Reply:
Done :)